core/imageroot/var/lib/nethserver/node/update-core.d/50start_promtail

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Since we are on beta1, we could also remove this code and document a manual step.